components/post_view/reaction/reaction.jsx

```diff
@@ -10,88 +10,152 @@ import OverlayTrigger from 'components/overlay_trigger';
 
 import * as Utils from 'utils/utils.jsx';
 
-export default class Reaction extends React.PureComponent {
-    static propTypes = {
+import './reaction.scss';
 
-        /*
-         * The post to render the reaction for
-         */
+export default class Reaction extends React.Component {
+    static propTypes = {
         post: PropTypes.object.isRequired,
-
-        /*
-         * The user id of the logged in user
-         */
         currentUserId: PropTypes.string.isRequired,
-
-        /*
-         * The name of the emoji for the reaction
-         */
         emojiName: PropTypes.string.isRequired,
-
-        /*
-         * The number of reactions to this post for this emoji
-         */
         reactionCount: PropTypes.number.isRequired,
-
-        /*
-         * Array of users who reacted to this post
-         */
         profiles: PropTypes.array.isRequired,
-
-        /*
-         * The number of users not in the profile list who have reacted with this emoji
-         */
         otherUsersCount: PropTypes.number.isRequired,
-
-        /*
-         * Array of reactions by user
-         */
         reactions: PropTypes.arrayOf(PropTypes.object).isRequired,
-
-        /*
-         * True if the user has the permission to add a reaction in this channel
-         */
         canAddReaction: PropTypes.bool.isRequired,
-
-        /*
-         * True if user has the permission to remove his own reactions in this channel
-         */
         canRemoveReaction: PropTypes.bool.isRequired,
-
-        /*
-         * The URL of the emoji image
-         */
         emojiImageUrl: PropTypes.string.isRequired,
-
         actions: PropTypes.shape({
-
-            /*
-             * Function to add a reaction to a post
-             */
             addReaction: PropTypes.func.isRequired,
-
-            /*
-             * Function to get non-loaded profiles by id
-             */
             getMissingProfilesByIds: PropTypes.func.isRequired,
-
-            /*
```

Comment:
Why don't we keep the comments here?

Reply:
I can, just removed for convenience when I was working. But I do wonder what our standard is regarding commenting on default props. Some have everything commented, others have nothing 🤷‍♂️